usr/share/lib/img_proof/tests/conftest.py

```diff
@@ -152,21 +152,31 @@ def is_suma_server(host, get_baseproduct, get_suma_version):
     def f():
         suma_server_product = '/etc/products.d/SUSE-Manager-Server.prod'
         suma_server = host.file(suma_server_product)
+
+        # SUMA >=5.1 renamed to Multi-Linux Manager
+        if not suma_server.exists:
+            suma_server_product = \
+                '/etc/products.d/Multi-Linux-Manager-Server.prod'
+            suma_server = host.file(suma_server_product)
+
```

Comment:
Since we are searching for known files I think a for loop is in order.

```
suma_server_product = ''
for prod_file in ('/etc/products.d/SUSE-Manager-Server.prod', '/etc/products.d/Multi-Linux-Manager-Server.prod'):
    suma_server = host.file(prod_file)
    if suma_server:
       suma_server_product = prod_file
       break
```

Reply:
Agree